### PR TITLE
Cerebral proxy

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

**Comment by Guria** [comment]
.cache must be ignored

**Comment by christianalfoni** [comment]
Okay, so I did some changes to how we organize:

1. By default there are no "devDependencies" in packages anymore, all is installed on root. There was tons of duplicates
2. The only reason to use "devDependencies" in a package is for specific dev dependencies for that package. For example parcel plugins (required to be in package), conflicting types like jest and mocha
3. There is a new script type called "setup". If you insert this script in a package it will run when the root `npm run setup` is run. This allows you to prepare stuff inside the package. For example installing the devDependencies required specifically for that package

Cleaned typing, errors etc. Now demos should just work out of the box etc.

File: .eslintrc.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   },
   "overrides": [
     {
-      "files": ["*.ts", "*.tsx"],
+      "files": ["*.ts", "*.tsx", "*.d.ts"],
```
Comment (henri-hulski) [inline .eslintrc.json]: `"*.d.ts"` is not needed as it's already included in `"*.ts"`

**Comment by henri-hulski** [comment]
Forgot that this is a older branch. So everything is okay.
Just one minor adjustment above.

**Comment by christianalfoni** [comment]
Cool, done! :)

**Pull request merged**